apps/client/src/hooks/application/use-application.ts

```diff
@@ -0,0 +1,33 @@
+import { useNavigate } from 'react-router-dom';
+import { useDismissAllNotifications } from '~/hooks/notifications/use-dismiss-all-notifications';
+import { useSetNavigationVisibility } from './use-set-navigation-visibility';
+
+/** Use to control the application. */
+export function useApplication() {
+  const navigateClient = useNavigate();
+  const dismissNotifications = useDismissAllNotifications();
+  const setNavigationVisibility = useSetNavigationVisibility();
+
+  function closeNavigation() {
+    setNavigationVisibility(false);
+  }
+
+  /**
+   * Navigate the client to a new page.
+   *
+   * @regards
```

Comment:
I haven't seen this tag before. Is it similar to `@remarks` in tsdoc?

Reply:
Oof, it should be `@remarks`!